### ISSUE TITLE
I'm interested in your yaml includes

### ISSUE THREAD

**Issue by mfisher87** [robotsandcake/homeassistant#1]
Hey, I've been eyeballing this repo since your hacker news post. I think it will help inspire more automation ideas for me. I really appreciate your willingness to share! However, I noticed that your configuration.yaml includes files which are not part of the repo:

```
cloud: !include cloud_component.yaml
group: !include groups.yaml
automation: !include automations.yaml
script: !include scripts.yaml
light: !include light.yaml
zone: !include zone.yaml
switch: !include switch.yaml
```

Could you push those up if you're willing to share them as well? I would also be interested in seeing a picture of your dashboard, despite the mess, if you're willing to share.

Thanks again! And extra thanks for including information about the hardware you're using. :heart:

**Comment by robotsandcake** [comment]
Absolutely, it has been one of those days today and I had just the second settled down to sort the repository out.I will start with the scripts and automation is  As they are the best documented I think.

  Is anything in particular you wanted me to focus on?

**Comment by robotsandcake** [comment]
Sorry, I accidentally closed this before seeing if you have finished!  How rude!

**Comment by mfisher87** [comment]
Awesome, thanks so much for the response! I thought perhaps you'd forgotten some files, but I am probably projecting my own absent-mindedness there :stuck_out_tongue: 

I'm always curious about other people's home automation setups in general, because sometimes you don't know what you want until you see what other people are doing. So nothing specific I'm interested in, just looking for inspiration in general. Thanks again for your time!

**Comment by robotsandcake** [comment]
@mfisher87 I am mainly uploading the files one at a time making sure that I sanitise them properly so I don't accidentally upload passwords and the like!  :-)

**Comment by robotsandcake** [comment]
@mfisher87 Wow I hate Git  with a fiery passion! 

It's probably going to be easier  For you to look me up on Keybase  so I can just explain anything you want to know! Heeee

Pushing an update in 10 minutes. I really need some help with static repositories like this without making glaring mistakes and uploading the keys to the kingdom, surely there must be some automated tools for this sort of thing?

I'm a home automation specialist Jim, not a Git specialist!

(Bad Star Trek reference is bad....)
